pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -408,25 +427,27 @@ func (s *Summarizer) getControlsFilePath(filename string) string {
 }
 
 func (s *Summarizer) getNodeTypeControlsFileMapping() map[string]NodeType {
-	// load node type files first
-	filepaths := map[string]NodeType{
-		s.getControlsFilePath(MasterControlsFilename): NodeTypeMaster,
-		s.getControlsFilePath(EtcdControlsFilename):   NodeTypeEtcd,
-		s.getControlsFilePath(NodeControlsFilename):   NodeTypeNode,
-	}
-	allFiles, err := filepath.Glob(fmt.Sprintf("%s/%s/*.yaml", s.ControlsDirectory, s.BenchmarkVersion))
-	if err != nil {
-		logrus.Errorf("error globing files: %v", err)
-		return filepaths
-	}
-	for _, f := range allFiles {
-		if controlFilesToIgnore[f] {
+	var filepaths = make(map[string]NodeType)
+	requiredFiles := s.BenchmarkToConfigMap[s.BenchmarkVersion]
+	for _, f := range requiredFiles {
+
+		if "master" == f {
```

Comment:
define const values for strings "master"/"etcd"/"node" etc and use const for comparison

Reply:
We have made the changes as suggested and updated the PR